cmd/helpMessages.go

```diff
@@ -30,100 +30,122 @@ Copies source data to a destination location. The supported directions are:
 
-Download a single file using OAuth authentication. Please use 'azcopy login' command first if you aren't logged in yet:
+Download a single file by using OAuth authentication. If you have not yet logged into AzCopy, please use azcopy login command before you run the following command.
+
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/blob]" "/path/to/file.txt"
 
-Download a single file with a SAS:
+Download a single file by using a SAS token:
+
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/blob]?[SAS]" "/path/to/file.txt"
 
-Download a single file with a SAS using piping (block blobs only):
+Download a single file by using a SAS token and then piping the output to a file (block blobs only):
+  
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/blob]?[SAS]" > "/path/to/file.txt"
 
-Download an entire directory with a SAS:
+Download an entire directory by using a SAS token:
+  
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/directory]?[SAS]" "/path/to/dir" --recursive=true
 
-A note about wildcards in URLs:
+A note about using a wildcard character (*) in URLs:
 
-The only usage of a wildcard in a URL that is supported is the final, trailing /*, and in the container name when not specifying a blob name.
-If you happen to use the character * in the name of a blob, please manually encode it to %2A to avoid it being treated as a wildcard character.
+There's only two supported ways to use a wildcard character in a URL. 
+You can use one just after the final forward slash (/) of a URL to obtain all of the files in a directory. 
+You can also use one in the name of a container as long as the URL refers only to a container and not to a blob.   
 
-Download the contents of a folder directly to the destination (rather than under a sub-directory):
- - azcopy cp "https://[srcaccount].blob.core.windows.net/[container]/[path/to/folder]/*?[SAS]" "/path/to/dir"
+Download the contents of a directory without copying the containing directory itself.
+ 
+  - azcopy cp "https://[srcaccount].blob.core.windows.net/[container]/[path/to/folder]/*?[SAS]" "/path/to/dir"
 
-Download an entire storage account at once
- - azcopy cp "https://[srcaccount].blob.core.windows.net/" "/path/to/dir" --recursive
+Download an entire storage account.
 
-Download an entire storage account at once with a wildcarded container name
- - azcopy cp "https://[srcaccount].blob.core.windows.net/[container*name]" "/path/to/dir" --recursive
+  - azcopy cp "https://[srcaccount].blob.core.windows.net/" "/path/to/dir" --recursive
+
+Download an entire storage account by using a wildcard symbol (*) in the container name.
+
+  - azcopy cp "https://[srcaccount].blob.core.windows.net/[container*name]" "/path/to/dir" --recursive
+
+Copy a single blob to another blob by using a SAS token.
 
```

Comment:
"by using" -> "using"

Also consider noting slightly above that Azure S2S transfers require a SAS token on the source for now.

Reply:
"by using" is actually better. Sometimes to keep titles succinct we'll omit the "by", but that's only to keep longer titles from truncating in search results or from using two lines in a TOC entry. Using this specific sentence as an example, omitting "by" leads the reader to infer that the "blob uses a SAS token". It doesn't really. The person who is typing the command applies a SAS token to the URL. Therefore, the task "copying a single blob to another blob" is performed "by using" a SAS token. This is why I've been replacing instances of "using" with "by using".